eng/Versions.props

```diff
@@ -18,6 +18,12 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
   </PropertyGroup>
+  <!-- Restore feeds -->
+  <PropertyGroup Label="Restore feeds">
+    <!-- In an orchestrated build, this may be overridden to other Azure feeds. -->
+    <DotNetAssetRootUrl Condition="'$(DotNetAssetRootUrl)'==''">https://dotnetbuilds.blob.core.windows.net/public/</DotNetAssetRootUrl>
```

Comment:
@alexperovich I thought you had implemented this already?

Reply:
I thought I did. Where is this coming from? It looks like maestro added this?

Reply:
This clearly looks like something maestro added, but I can't see anywhere in the code that would cause this.

Reply:
O, wait, I can see where it was done manually now. What error was produced that implied that this was needed? All I can see is a test failure.

Reply:
I can't even see where this property is used. From what I can tell no file in main uses this property at all.

Reply:
I added that to mimic a change we made in dotnet/aspnetcore (to flow a runtime update). But, it hasn't changed the errors seen in validation builds. It also appears the property affects custom infrastructure e.g. installer builds that we don't have in this repo.

Reply:
Please let me know if this should be reverted @mmitche @alexperovich